guides/16_Video_Layout.md

```diff
@@ -0,0 +1,527 @@
+
+#### Uplink Policy
+
+The video encoding parameters are determined by the uplink policy, which takes into consideration the number of video publishers on the call. If necessary, it may suggest encoding multiple “simulcast” layers (with different bitrates and resolutions). The JavaScript SDK currently provides two uplink policies:
+
+**NScaleVideoUplinkBandwidthPolicy (Default):** In this policy, an attendee encodes and sends one video layer upstream and scales the resolution and bitrate based on the number of video publishers. This is to ensure the receiving attendees who are subscribed to these videos do not experience excessive network or compute load, which may degrade the meeting quality.
+
+**SimulcastUplinkPolicy:** Each attendee may encode and send up to two layers of video with different bitrates and resolutions under this policy. It enables the recipient to switch between the two video layers to adapt to changing network conditions or different layouts (e.g., a featured video may desire a higher resolution than a gallery view video).
+
+- Note: the `SimulcastUplinkPolicy` is currently only supported in Chrome 76 and above versions.
+
+#### Downlink Policy
+
+The downlink policy controls which remote video sources are received downstream based on the network condition, local resources (e.g., CPU), and other factors such as layout changes in the application. The JavaScript SDK currently provides two downlink policies:
+
+**AllHighestVideoBandwidthPolicy (Default):** This policy subscribes to the highest quality video layer of all publishers. Under constrained network, some or all videos may freeze as packets are lost on the network, and they will recover automatically once network congestion is reduced.
+
+**VideoPriorityBasedPolicy:** This policy allows builders to choose and configure which video sources to subscribe to, their relative priorities and preferences such as, `targetDisplaySize` . Under constrained network, the policy will automatically pause video tiles based on the priority of the video sources until recovery. If used with the `SimulcastUplinkPolicy`, the policy may temporarily degrade to lower resolutions. For videos of the same priority, the policy first tries to allocate bandwidth to ensure as many videos can be displayed as possible before upgrading to higher quality if the bandwidth allows.
+
+## Recommendations for Common Layouts
+
+### Case 1: Small Grid Layout (up to 4 Videos) with Fixed Tile Size
+
+We recommend the default configuration of the `NScaleVideoUplinkBandwidthPolicy`  and  `AllHighestVideoBandwidthPolicy` when displaying a small number of video tiles of a fixed size, so the recipient is less likely to get overwhelmed. Video freezes can occur due to network congestion and recover automatically.
+
+#### JavaScript SDK Implementation
+
+Refer to code sample in [Use case 16](https://github.com/aws/amazon-chime-sdk-js#video).
+
+#### React Components Library Implementation
```

Comment:
Is this something you will move or add to the react component library documentation?